src/main/java/kr/codesqaud/cafe/repository/UserDao.java

```diff
@@ -26,23 +26,28 @@ public void addUser(User user) {
     }
 
     public List<UserDto> findUserAll() {
-        String sql = "SELECT * FROM USER";
+        String sql = "SELECT * FROM CAFEUSER";
         return jdbcTemplate.query(sql, new BeanPropertyRowMapper<>(UserDto.class));
     }
 
     public List<UserDto> findUserByUserId(String userId) {
```

Comment:
```suggestion
    public Optional <UserDto> findUserByUserId(String userId) {
```
Id 조회는 결과가 1 or 0 이니까 list보다 Optional이 좋지 않을까요?